platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gui/SwirldsGui.java

```diff
@@ -44,16 +43,16 @@ private SwirldsGui() {}
      * Create a new window with a text console, of the recommended size and location, including the Swirlds menu.
      *
      * @param platform the platform to create the console with
+     * @param winNum the number of the window, starting at 0
      * @param visible  should the window be initially visible? If not, call setVisible(true) later.
      * @return the new window
      */
-    public static Console createConsole(final Platform platform, final int winNum, final boolean visible) {
+    public static Console createConsole(@NonNull final Platform platform, final int winNum, final boolean visible) {
```

Comment:
Minor: there is no null-check for platform

Reply:
The platform is accessed in the method. we've been leaving off null checks because a null pointer exception would be generated in the method that the argument is passed.   We add null checks when the null pointer exception is not generated in the same call stack.  (Setting local variables that get used later).